packages/nextjs/app/admin/page.tsx

```diff
@@ -182,7 +182,9 @@ const AdminPage = () => {
         <div className="flex flex-col lg:flex-row gap-12 mt-4">
           <div className="p-8 bg-warning/5 lg:w-1/2">
             <div className="flex justify-between items-center">
-              <h2 className="font-bold text-xl">New Grant Proposals</h2>
+              <h2 className="font-bold text-xl">
+                New Grant Proposals {newGrants && newGrants.length > 0 && `(${newGrants.length})`}
```

Comment:
It's a personal thing, but Instead of `(${newGrants.length})` I'd have gone with `<>({newGrants.length})</>`. No need to change it, just mentioning it :P

Reply:
Since I was pushing extra code, refactored this also, it's indeed easier to read! 🙌